catalog.html

```diff
@@ -62,19 +62,47 @@
             </svg>
             <span class="visually-hidden">Поиск.</span>
           </button>
-          <input type="text">
+          <form>
+            <div class="popover-search">
+              <label class="visually-hidden" for="search">Поиск по сайту.</label>
+              <input class="field field-search" type="search" id="search" name="search" placeholder="Поиск по сайту">
+            </div>
+          </form>
```

Comment:
Форма у нас получается нулевая + немного странная логика получается, что `popover-search` позиционируется, скрывается и тд, а форма так, в сторонке где-то еще и пустая. Поэтому я бы сделал так:
```suggestion
          <div class="popover-search">
            <form>
              <label class="visually-hidden" for="search">Поиск по сайту.</label>
              <input class="field field-search" type="search" id="search" name="search" placeholder="Поиск по сайту">
            </form>
          </div>
```